index.html

```diff
@@ -152,7 +154,7 @@ <h4>Featured Speakers</h4>
       </section>
       <section class="desktop" id="partners">
         <div class="section-heading">
-          <h4>Partner</h4>
+          <h4>Partners</h4>
```

Comment:
- As was requested by the previous reviewer, kindly change the line color to red, under `Partner`, `Main Program`, and under `Featured Speakers` to match with the Figma design

![image](https://github.com/badger-99/Capstone-Project-1/assets/93630700/e69e0f5b-12e7-402d-97a3-cd71f17def11)

![image](https://github.com/badger-99/Capstone-Project-1/assets/93630700/4f5a53db-584e-4e11-989c-8cb7422539f3)

![image](https://github.com/badger-99/Capstone-Project-1/assets/93630700/9cd9795f-d488-4578-8c1c-cf4bac98e379)